Tests/Requests/Controller_RequestController_AddRequest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Controller.DbControllers;
+using Model.Database.Contexts;
+using Model.DbModels;
+using NUnit.Framework;
+
+namespace Tests.Requests
+{
+    [TestFixture]
+    public class Controller_RequestController_AddRequest
+    {
+        private MockDatabaseContext context;
+        private RequestController requestController;
+        private Request testRequest;
```

Comment:
Zelfde geld hiervoor, je kunt in je testen gewoon ID = 1 gebruiken, ipv `testRequest.ID`
En dan kun je ook je hele TearDown weghalen :)

Reply:
Bij line 25 wordt er een item gemaakt, ik dacht dat daarvoor een teardown nodig was?

Reply:
Neuh, niet in een mockdb.

Reply:
OKEY

Reply:
betekent dat de teardown van de andere test ook overbodig is?

Reply:
Ligt eraan hoe ze gebouwd zijn haha

Reply:
Ok!